core/src/test/java/hudson/FilePathTest.java

```diff
@@ -349,8 +353,31 @@ private FilePath createFilePath(final File base, final String... path) throws IO
         final String filePrefix = "JENKINS-10629";
         checkTarUntarRoundTrip(filePrefix, largeFileSize);
     }
-     
+
+    private void checkUntar2Path(String filePrefix, long fileSize) throws IOException, InterruptedException {
```

Comment:
I think this test is about `archive2`, not `untar2`, right?

```suggestion
    private void archive2ReturnsPaths(String filePrefix, long fileSize) throws IOException, InterruptedException {
```